src/client.toit

```diff
@@ -40,6 +42,36 @@ class Client:
       max_qos := 1
       subscription_callbacks_.set topic (CallbackEntry_ callback max_qos)
 
+  /**
+  Variant of $(constructor --transport) that connects to the given $host:$port over TCP.
+  */
+  constructor
+      --host/string
+      --port/int=1883
```

Comment:
The types, =, and default values have varying spaces around them.

Reply:
more consistent now.